src/modules/70-pipeline/components/RunPipelineModal/VisualView.tsx

```diff
@@ -261,6 +266,9 @@ export interface PipelineInputSetFormWrapperProps {
   showDivider?: boolean
   listOfSelectedStages?: string[]
   isRetryFormStageSelected?: boolean
+  repoIdentifier?: string
```

Comment:
```suggestion
  repoName?: string
```

then use the same Interface here as well

Reply:
Will cleanup in follow up PR